### ISSUE TITLE
QuickLook Interactions - hopping asset does not move in ARQL like in needle 

### ISSUE THREAD

**Issue by PixelPartner** [needle-tools/needle-engine-samples#10]
The "Move to" behaviour differs, because the moving prims are way over in the "Animation" section instead of moving between their dedicated trigger/target-assets.

**Comment by hybridherbst** [comment]
Good catch :) the "Move to" actions are behaving weird in QuickLook, my assumption is that Reality Composer builds complex hierarchy chains to counter some bugs in the underlying implementation.

**Comment by PixelPartner** [comment]
I assume it's related to the offset introduced by the group transform of the "Move to" group
